packages/frontity-org-theme/src/components/styles/global-styles.tsx

```diff
@@ -80,40 +78,55 @@ const documentSetup = (colors: FrontityOrg["state"]["theme"]["colors"]) => css`
   img {
     max-width: 100%;
   }
+  div#root > div > div {
```

Comment:
I would use a class name here instead of this selector, but anyway, let's do this and just change this in the future if needed.